repology/parsers/parsers/nix.py

```diff
@@ -177,7 +177,13 @@ def iter_parse(self, path: str, factory: PackageFactory) -> Iterable[PackageMake
                 if re.match('[0-9a-f]*[a-f][0-9a-f]*$', pkg.version) and len(pkg.version) >= 7:
                     pkg.set_flags(PackageFlags.IGNORE)
 
-                pkg.add_homepages(meta.get('homepage'))
+                pkg.add_links(LinkType.UPSTREAM_HOMEPAGE, meta.get('homepage'))
+
+                if 'downloadPage' in meta:
```

Comment:
This is not correct, `UPSTREAM_DOWNLOAD` is for direct links to tarballs, not to a download page, you'll probably need to remove this. Otherwise good.

Reply:
Removed